etc/openqa/settings_ui_links.ini

```diff
@@ -0,0 +1,3 @@
+[links]
+#SCHEDULE=schedule
+#AUTOYAST=autoyast_sle15
```

Comment:
I recommend against a separate config file. Can you just keep it in one? Otherwise you also need to handle installing this file in package installations, e.g. Makefile and spec-file.

Please all sle-specific versions, even if just for an example. I would prefer to have version-agnostic code only and every other versions becomes old sooner than we think :)

Reply:
Although i tried to take care of the package installation, some things are not clear to me. i hope it looks better to you now.